app/sliders/mapping.py

```diff
@@ -45,4 +45,10 @@
     "Umsatz mit Marktneu-heiten in Mio. €": "umsatz_mio_markt",
     "Innovations-ausgaben in Mio. €": "innovations_ausgaben_mio",
     "FuE-Ausgaben in Mio. €": "fue_ausgaben_mio",
-}
\ No newline at end of file
+}
+
```

Comment:
If we use the list of branches from the config, this mapping list is not necessary. We can delete it.